src/main/scala/scala/tools/partest/nest/Runner.scala

```diff
@@ -759,7 +761,7 @@ class SuiteRunner(
     // |Java Classpath:             ${sys.props("java.class.path")}
   }
 
-  def onFinishTest(testFile: File, result: TestState): TestState = result
+  def onFinishTest(testFile: File, result: TestState): TestState = { unused(testFile) ; result }
```

Comment:
This is also wrong, imho..

Reply:
@dwijnand do you prefer pre or post syntax? an annotation to ignore or a reporter that knows to ignore it? This is a middle solution.

Reply:
I prefer the pre-syntax. IMO the unused param checker shouldn't warn on non-effectively final methods.

(Sorry I'm a little salty atm as I really want to use "warn unused"/Xlint in sbt/sbt but I've been dealing with lots and lots of these..)

Reply:
Finding the sweet spot for usability is hard, so input is appreciated. Warning on params was always assumed to be dicey. It sounds like: warn on final, unless asking for non-strict. Then, let me disable it with an annotation if I want.

I proposed this solution here because there's no extra machinery.

Reply:
Btw another false positive (ish) is enriching objects, e.g `implicit def enrichFoo(f: Foo.type)`.